Examples/openai-agents-sdk/README.md

```
@@ -0,0 +1,63 @@
+- Python 3.8+
+- Nebius API key
+- Resend API key
+- A verified sender email for Resend
+
+## Environment Variables
+
+The application requires the following environment variables:
+
+- `NEBIUS_API_KEY`: Your Nebius Studio API key
+- `RESEND_API_KEY`: Your Resend API key
+- `EXAMPLE_BASE_URL`: Nebius API base URL (defaults to "https://api.studio.nebius.ai/v1")
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)